README.md

```diff
@@ -31,6 +31,12 @@ Aerobridge is fully open source and you can get involved by participating in our
 
 This is an open source implementation of the "Manufacturer's Management Server" to help with key signing and managing interactions with India's [Digital Sky](https://digitalsky.dgca.gov.in/) API infrastructure. This server can be deployed to any public and private cloud and be used to manage communication with DGCA's Digital Sky Infrastructure and manage NPNT permissions. For a more technical introduction, see this [presentation](https://docs.google.com/presentation/d/1cZrNwNrLtLIj5eKEGql2HN-G1gZFbbGhGbiTB1i16So/edit?usp=sharing).
 
+## Dependencies
+``` 
+sudo apt install libpq-dev          # Postgre SQL Database
```

Comment:
Is there any way that we can use SQLite?  I guess I am asking if pq-dev is necessary for the testing?

Reply:
Yeah, sqlite3 would work. Initially, while setting up postgre came as a result which would support sqlite3 too inherently, that's why I went with it. Have updated the requirements now.